src/pages/user-cvs/user-cvs.tsx

```diff
@@ -1,9 +1,10 @@
 import { useUserBreadcrumbs } from "@/modules/users/hooks";
+import CvsList from "@/modules/users/ui/cvs-list";
 
 const UserCVs = () => {
-    useUserBreadcrumbs("cvs");
```

Comment:
No need to remove that arg, since "CVs" tab should also generate an additional breadcrumb item based on it (the only tabs that shouldn't are usually the 1st (and default) ones, like "Profile" for users & "Details" for CVs)
![image](https://github.com/savelievamareena/cv-final/assets/99599598/df51410f-c4fe-489f-b499-2b5d5e4c4bef)